tests/data/test_particle.py

```diff
@@ -9,21 +9,24 @@
     compute_gellmann_nishijima,
```

Comment:
Its a bit confusing that we have two, `test_particle.py` files. Would be good to merge them

Reply:
Eventually, yes. But this is related to #197. For now the `tests` folders mirror the structure of the submodules.